core/modfunc.go

```diff
@@ -376,7 +376,7 @@ func moduleAnalyticsProps(mod *Module, prefix string, props map[string]string) {
 		git := source.AsGitSource.Value
 		props[prefix+"source_kind"] = "git"
 		props[prefix+"git_symbolic"] = git.Symbolic()
-		props[prefix+"git_clone_url"] = git.CloneURL // todo(guillaume): remove as deprecated
+		props[prefix+"git_clone_url"] = git.CloneRef // todo(guillaume): remove as deprecated
 		props[prefix+"git_clone_ref"] = git.CloneRef
```

Comment:
This is used for analytics? When can we delete it (according to the node, @grouville)?

Reply:
FYI @aluzzardi